### PR TITLE
add new aws recommendations

### PR DESCRIPTION
**Pull request by joeyouss**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Review by nikunj-badjatya — DISMISSED** [review]

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md [hunk 1]
```diff
@@ -215,3 +215,187 @@ policies:
...
+          Id: harness-default
+          Status: Enabled
+          Tierings:
+            - Days: 90
+              AccessTier: ARCHIVE_ACCESS
+            - Days: 180
+              AccessTier: DEEP_ARCHIVE_ACCESS
+```
+
+**Savings Computed:**
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md:326]: Missing Savings Computed.

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md [hunk 2]
```diff
@@ -215,3 +215,187 @@ policies:
...
+            Expiration:
+              ExpiredObjectDeleteMarker: True
+            AbortIncompleteMultipartUpload:
+              DaysAfterInitiation: 7
+            NoncurrentVersionExpiration:
+              NoncurrentDays: 30
+              NewerNoncurrentVersions: 6
+```
+
+**Savings Computed**:
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md:285]: Missing Savings Computed.

**Review by iamanmolm — CHANGES_REQUESTED** [review]

**Review by iamanmolm — APPROVED** [review]

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md [hunk 3]
```diff
@@ -215,3 +215,259 @@ policies:
...
+Let's assume the following for a single S3 bucket:
+
+**Total Storage Used**: 1 TB (1,024 GB) in the S3 Standard storage class.
+
+**Storage Distribution:**
+- Current versions: 50% (512 GB)
+- Noncurrent versions: 40% (410 GB)
+- Incomplete multipart uploads: 10% (102 GB)
+
+**Without Lifecycle Policy**
```
Comment (iamanmolm) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md:307]: Can we keep it side by side maybe in a table comparison?

**Review by iamanmolm — CHANGES_REQUESTED** [review]

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md (hunk 3 above)
Comment (joeyouss) [reply docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md]: done

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md [hunk 4]
```diff
@@ -215,3 +215,242 @@ policies:
 
 ---
 
+### Recommendation: under-utilized-elasticcache-cluster
+**Description:** Delete underutilised cache cluster with CPU utilisation less than 5% in last 7 days.
```
Comment (brian-f-harness) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md:219]:
```suggestion
**Description:** Delete underutilized cache cluster with CPU utilization less than 5% in the last 7 days.
```

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md [hunk 5]
```diff
@@ -215,3 +215,242 @@ policies:
...
+        days: 7
+        period: 86400
+        value: 5
+        op: less-than
+    actions:
+      - type: delete
+        skip-snapshot: false
+```
+
+**Savings Computed:** The policy identifies a list of resources on which potential savings are calculated by summing up cost of each resource for last 30 days.
```
Comment (brian-f-harness) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md:241]:
```suggestion
**Savings Computed:** The policy identifies a list of resources on which potential savings are calculated by summing up the cost of each resource for the last 30 days.
```

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md [hunk 6]
```diff
@@ -215,3 +215,242 @@ policies:
...
+
+**Permissions Required:** 
+- **Dry Run:** 
+    - ```elasticache:DescribeCacheClusters```
+- **Run Once:** 
+    - ```elasticache:DescribeCacheClusters```
+    - ```elasticache:DeleteCacheCluster```
+    - ```elasticache:DeleteReplicationGroup```
+
+---
```
Comment (brian-f-harness) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md:251]:
```suggestion
```

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md [hunk 7]
```diff
@@ -215,3 +215,242 @@ policies:
...
+- [Managing S3 Costs](https://aws.amazon.com/blogs/storage/optimizing-costs-with-amazon-s3-lifecycle-configurations/)
+
+**Permissions Required:**
+- **Dry Run:** 
+    - ```s3:GetLifecycleConfiguration```
+- **Run Once:** 
+    - ```s3:GetLifecycleConfiguration```
+    - ```s3:PutLifecycleConfiguration```
+
+---
```
Comment (brian-f-harness) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md:326]:
```suggestion
```

**Review by brian-f-harness — DISMISSED** [review]
Added a few minor suggestions, otherwise LGTM

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md [hunk 8]
```diff
@@ -215,3 +215,242 @@ policies:
...
+- **Run Once:** 
+    - ```elasticache:DescribeCacheClusters```
+    - ```elasticache:DeleteCacheCluster```
+    - ```elasticache:DeleteReplicationGroup```
+
+---
+
+### Recommendation: s3-lifecycle-configuration
+
+**Description:** Configure lifecycle for s3 buckets wherever it is absent which would help to reduce storage spend
```
Comment (brian-f-harness) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md:255]:
```suggestion
**Description:** Configure lifecycle for S3 buckets wherever it is absent which would help to reduce storage spend
```

File: docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md [hunk 9]
```diff
@@ -215,3 +215,242 @@ policies:
...
+    - ```s3:GetLifecycleConfiguration```
+- **Run Once:** 
+    - ```s3:GetLifecycleConfiguration```
+    - ```s3:PutLifecycleConfiguration```
+
+---
+
+### Recommendation: s3-intelligent-tiering-configuration
+
+**Description:** Configure intelligent tiering for s3 buckets wherever it is disabled which would help to reduce storage spend.
```
Comment (brian-f-harness) [inline docs/cloud-cost-management/5-use-ccm-cost-governance/asset-governance/AWS/AWS-recommendations.md:330]:
```suggestion
**Description:** Configure intelligent tiering for S3 buckets wherever it is disabled which would help to reduce storage spend.
```

**Pull request merged**